addons/l10n_mr/i18n/fr.po

```diff
@@ -0,0 +1,156 @@
+"Plural-Forms: \n"
+
+#. module: l10n_mr
+#: model:account.report.column,name:l10n_mr.tax_report_tax_16
+msgid "16%"
+msgstr "16%"
+
+#. module: l10n_mr
+#: model:account.report.column,name:l10n_mr.tax_report_tax_18
+msgid "18%"
+msgstr "18%"
+
+#. module: l10n_mr
+#: model:ir.model,name:l10n_mr.model_account_chart_template
+msgid "Account Chart Template"
+msgstr "Modèle de plan comptable"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_base
+msgid "Calculation of VAT base"
+msgstr "Calcul de la base de la TVA"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_net
+msgid "Calculation of net VAT"
+msgstr "Calcul de la TVA nette"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_base_completed_operations
+msgid "Completed Operations"
+msgstr "Opérations Réalisées"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_deductions_prev_month
+msgid "Credit carried forward to previous month"
+msgstr "Crédit reporté du mois précédent"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_net_carry_forward
+msgid "Credit to be carried forward (line 16 - line 15)"
+msgstr "Crédit reporté sur mois précédent"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_net_detuctable
+msgid "Deductible VAT (line 14)"
+msgstr "TVA décutible (ligne 14)"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_deductions_total
+msgid "Deductible VAT (lines 10 to 13)"
+msgstr "TVA deductible (lignes 10 à 13)"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_deductions
+msgid "Deductions"
+msgstr "Déductions"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_base_exports
+msgid "Exports"
+msgstr "Exportations"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_gross
+msgid "Gross VAT"
+msgstr "TVA brute"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat
+msgid "Gross VAT calculation"
+msgstr "Calcul de la TVA brute"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_vat_net_total
+msgid "Net VAT (line 15 - line 16)"
+msgstr "TVA nette à payer (ligne 15 - ligne 16)"
+
+#. module: l10n_mr
+#: model:account.report.column,name:l10n_mr.tax_report_non_taxables
+msgid "Non-Taxables"
+msgstr "Non taxables"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_deductions_vat_other
+msgid "Other deductions to be justified"
+msgstr "Autres dédiuctions à justifier"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_base_sales
+msgid "Sales"
+msgstr "Ventes"
+
+#. module: l10n_mr
+#: model:account.report.line,name:l10n_mr.tax_report_base_services
+msgid "Services"
+msgstr "Prestations de service"
+
+#. module: l10n_mr
+#: model:account.report,name:l10n_mr.tax_report
+msgid "Tax Report"
+msgstr "Tax Report"
```

Comment:
This is not French. "Rapport de taxes" maybe literally, or maybe there is another name for it?